cfme/infrastructure/config_management.py

```diff
@@ -185,13 +185,14 @@ def delete(self, cancel=False, wait_deleted=True, force=False):
         navigate_to(self, 'All')
         sel.check(Quadicon(self.quad_name, None).checkbox())
         item_text = version.pick({'5.6': 'Remove selected items from the VMDB',
-                                  '5.7': 'Remove selected items',
                                   version.LATEST: 'Remove selected items'})
         cfg_btn(item_text, invokes_alert=True)
 
         sel.handle_alert(cancel)
         if not cancel:
-            flash_msg = 'Delete initiated for 1 provider'
+            flash_msg = version.pick({'5.6': 'Delete initiated for 1 provider',
+                                      version.LATEST: 'Delete initiated for 1 Provider'})
```

Comment:
I used LATEST in order to get rid of changing this again when we start testing 5.8+. are you still certain about changing this ?